🐛 [BUG] - Local setup fails: missing greenhouse-webhook Deployment causes webhook connection refused

### Priority

None

### Description

When running Greenhouse locally with `$ PLUGIN_DIR=[...] make setup` the process fails because the Kubernetes API server cannot reach the mutating webhook. No pods are backing the `greenhouse-webhook-service`.


### Reproduction steps

```bash
1. Run local setup with the plugin directory:
   
   PLUGIN_DIR=… make setup
   
3. Observe failure during the “samples” step:
   
   Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook 
   "morganization.kb.io": failed to call webhook: Post "https://greenhouse-webhook- 
   service.greenhouse.svc:443/mutate-greenhouse-sap-v1alpha1-organization?timeout=10s": dial tcp 
   10.96.27.137:443: connect: connection refused
   make: *** [samples] Error 1
   
4. Attempt to apply plugin definitions:
   
   kubectl apply -f plugindefinition.yaml
   Error from server (InternalError): failed calling webhook "mplugindefinition.kb.io":
   dial tcp 10.96.27.137:443: connect: connection refused
   
5. Inspect resources in the greenhouse namespace:
 - `greenhouse-webhook-service` exists but has no endpoints
 - No greenhouse-webhook Deployment is present
```

### Manifests

```yaml

```

### Screenshots

```bash

```

--- a/internal/local/commands/setup.go
+++ b/internal/local/commands/setup.go
@@ -90,6 +90,10 @@ func processDevSetup(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
+		// re-write the config file with the hostPath config
+		if hostPathConfig != "" {
+			cfg.Cluster.ConfigPath = hostPathConfig
+		}
 		env := setup.NewExecutionEnv().
 			WithClusterSetup(cfg.Cluster)
 		for _, dep := range cfg.Dependencies {

